### PR TITLE
Update topaz_cli.md

### PR DESCRIPTION
**Pull request by ratishkr**
Updated Topaz CLI.md with TED CLI section.

### CODE REVIEW

**Comment by ratishkr** [comment]
Hi @steve-kansa 
Please assign this PR for review. and let me know if there is anything I need to do further. 
This should only be merged when 20.06.01 is GA. Thanks.

File: docs/apis/topaz_cli.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ These are
 `CodeCoverageCLI.bat` | Allows downloading results from a Xpediter Code Coverage repository by specifying the repository name, system name and test ID.
 `IspwCLI.bat` | Used for the Git to ISPW integration
 `SCMDownloaderCLI.bat` | Allows interaction with the ISPW downloader to download sources from ISPW repositories, Endevor repositories or PDS 
+`SCMDownloaderCLI.bat` | Allows interaction with the ISPW downloader to download sources from ISPW repositories, Endevor repositories or PDS
```
Comment (bszydlow) [inline docs/apis/topaz_cli.md:20]: Ratish, this looks like this line were duplicated. There seem to be other changes, I think they are all whitespace changes but this is not really our help. Would be safer to make sure that we are just adding our part

File: docs/apis/topaz_cli.md [hunk 2]
```diff
@@ -933,4 +935,91 @@ Parameter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Abbr.  | Description
...
\ No newline at end of file
+`-save` | -s | Save output or not. Set to TRUE to write output, or FALSE for no output. When set to FALSE, no files will be created and the archive will not be updated. Defaults to TRUE.
+
+## Topaz Enterprise Data CLI (TedCLI.bat)
+
+The Topaz Enterprise Data CLI executes TED specifications such ConverterPro, ComparePro, Related Extract, Related Loader and Execution Suites. The specifications will have to be created using the relevant application client available in the File-AID/EX perspective of Topaz and the specifications should be saved and made available in any of the shared repositories. Moreover the shared repository should be configured on a standalone communication manager for using it with TED CLI. Further, to execute the specifications via the TED CLI, standalone execution server needs to be installed if not already available. Please note that the embedded communication manager and execution server that is available with the Topaz TED feature cannot be used for executing TED cli.
```
Comment (bszydlow) [inline docs/apis/topaz_cli.md:942]: Ted specifications including instead of such

File: docs/apis/topaz_cli.md [hunk 3]
```diff
@@ -933,4 +935,91 @@ Parameter&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Abbr.  | Description
...
+
+The standalone communication manager is installed along with the File-AID Services, and can be configured on Topaz Workbench preferences, under the TED section for use with the client. Once configured and set as current, the Repository Management utility shall be used to configure the shared repository on to the current communication manager. Please refer to the TED install guide for more information.
```
Comment (bszydlow) [inline docs/apis/topaz_cli.md:944]: in Topaz Workbench preferences

**Comment by ratishkr** [comment]
Hi Barb, 
I used Atom for working with the md file, it automatically trims the trailing spaces when the doc is saved. 
And the duplicate line occurred from is an attempt to discard the change. Fixed the other two comments as well and created a new pull request from a clean branch. I will close this PR. 

Thanks
Ratish

**Pull request closed without merging**